lib/Doctrine/Common/Collections/AbstractLazyCollection.php

```diff
@@ -274,7 +275,9 @@ public function slice($offset, $length = null)
     }
 
     /**
-     * {@inheritDoc}
+     * @return Traversable
```

Comment:
I think `@return Traversable<int|string, mixed>` can work

Reply:
it does, thank you!